satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -283,12 +288,17 @@ func rootMutation(service *console.Service, mailService *mailservice.Service, ty
 
 					var emailErr errs.Group
 					for _, user := range users {
+						userName := user.ShortName
+						if user.ShortName == "" {
+							userName = user.FullName
+						}
+
 						err = mailService.SendRendered(
 							p.Context,
-							[]post.Address{{Address: user.Email, Name: fmt.Sprintf("%s %s", user.FirstName, user.LastName)}},
+							[]post.Address{{Address: user.Email, Name: fmt.Sprintf("%s %s", user.FullName, user.ShortName)}},
```

Comment:
What is the meaning of combining Fullname and Shortname?

Reply:
not all cultures have the concept of first name and last name. the only general thing that works well across all cultures is full name and preferred short name

Reply:
`Name: fmt.Sprintf("%s %s", user.FullName, user.ShortName)}}`
I mean for email recipient name it should be one of the above but not both combined